src/core/computedvalue.ts

```diff
@@ -105,8 +105,7 @@ export class ComputedValue<T> implements IObservable, IComputedValue<T>, IDeriva
      * This is useful for working with vectors, mouse coordinates etc.
      */
     constructor(options: IComputedValueOptions<T>) {
-        if (process.env.NODE_ENV !== "production" && !options.get)
-            throw "[mobx] missing option for computed: get"
+        invariant(options.get, "missing option for computed: get")
```

Comment:
Let's keep `[mobx]` prefix there.

Reply:
https://github.com/mobxjs/mobx/blob/master/src/utils/utils.ts#L36-L38 adds the prefix